spec/support/shared_examples/request/suspended_orgs_permissions.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.shared_examples 'Permissions when organization is suspended' do
+  before do
+    org.status = 'suspended'
+    org.save
+  end
+
+  it_behaves_like 'permissions for single object endpoint', ALL_PERMISSIONS do
```

Comment:
It's possible to replace `ALL_PERMISSIONS` with something else so as to just test admin and space developer, as some of the other roles are less interesting. However that does add complexity.